docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst

```diff
@@ -143,12 +143,53 @@ Args syntax
      - TriggerName_
      - Name of the event trigger
 
+
+.. _redeliver_event:
+
+redeliver_event
+---------------
+
+``redeliver_event`` is used to redeliver an existing event. For e.g., if an event is marked as error (
+say it did not succeed event after retries) then you can redeliver it using this API. Note that this will reset the count of retries so far,
```

Comment:
```suggestion
say it did not succeed after retries), you can redeliver it using this API. Note that this will reset the count of retries so far.
```